Makefile

```diff
@@ -1,5 +1,4 @@
 BEAT_NAME=apm-server
-BEAT_DESCRIPTION=Elastic Application Performance Monitoring Server
```

Comment:
From what I saw this would be actually used in `mage`.  Where does the description now come from?